app/controllers/application_controller/ci_processing.rb

```diff
@@ -1609,42 +1607,55 @@ def scanclusters
     cluster_button_operation('scan', 'Analysis')
   end
 
+  def each_host(hosts, task_name)
```

Comment:
Prefer `host_ids` as the variable name so that it's clear it's not host objects, but their ids.